### DISCUSSION TITLE
Planning for the October 2023 release

### DISCUSSION THREAD

**Discussion by raman-m** [ThreeMammals/Ocelot#1771] [General]
@RaynaldM @ggnaegi 

I would like to understand the current release (October'23) scope of features. We will have [less features](../compare/main...develop) than previous [September'23](../releases/tag/20.0.0) release, but it is OK, because I was busy with docs release (#1755) for a 2 weeks.

Let's plan the rest of work.

@ggnaegi 
- #1745 
- #1753 
- no time to deliver more

@RaynaldM 
- #1714 
- Something else?

@raman-m From my side
- #1182 by @wast 
- and a few PRs by @jlukawska like
  - #1183
  - and maybe #1215 

Please send me a feedback ASAP.

**Comment by ggnaegi** [comment]
@raman-m ok, I understand for the request mapper improvement and the polling - the two PRs are big -, but maybe maybe this one could still fit in: https://github.com/ThreeMammals/Ocelot/pull/1769 "Bad error handling for 413 payload too large"?

**Reply by raman-m** [reply]
- #1769 

Hmm... The PR is linked to low priority bug. It is not reviewed at all... Have all tests? Need to update docs?

**Reply by raman-m** [reply]
Looks like you will have some time to polish it up, right?
Include in Nov'23 release?

**Comment by RaynaldM** [comment]
on my side, it is ok.
We're going to migrate to .net 8, I plan to migrate Ocelot to .net 8 in January (and I'll try to make it compatible with 6 and 7).

**Reply by raman-m** [reply]
I meant not future releases, I mean current October's release: the features we've merged to **develop**... 😄 
If you've prepared some PRs as ready for review, as a candidate to merge, then It is definetly right time to show it to the team...

----
>We're going to migrate to .net 8, I plan to migrate Ocelot to .net 8

Yes, we will... But .NET 8 will be released in a month officially, at the end of this November... It is impossible to do this migration in November's release, but it would be better to start migration as soon as possible. So, I believe we need to make migration and include this at least in December's release. But.... technically it is possible to make migration even for November's release.
We will discuss that... Now I'm focusing on current October's release trying to add more features, if possible.

**Reply by RaynaldM** [reply]
sorry for my bad english, I have said "I plan to migrate Ocelot to .net 8 in January"

**Reply by raman-m** [reply]
I've got it! ...in 2024

**Reply by ggnaegi** [reply]
.NET 8 will be released in 10 days, it's the new LTS version, we could try to push something before the end of the month and start the review process...

**Comment by ggnaegi** [comment]
@raman-m what about #1580?

**Reply by raman-m** [reply]
Linked PR #1666 has unit tests only. A few acceptance tests should be added.
Also, it seems, the docs must be reviewed and updated...
Ask the author? Maybe he will write acceptance tests, but he won't update docs... I explained him a lot how to move this PR forward...
I am afraid it is not ready for October's release... probably it will be included in November release.
More collaboration is required...

**Reply by raman-m** [reply]
Hi @ggnaegi !
Do you think that #1580 (#1666) is a good candidate for November release?

**Reply by ggnaegi** [reply]
@raman-m yes, it would be a good candidate for the release.

**Reply by raman-m** [reply]
Added it to Nov'23 release

**Comment by raman-m** [comment]
- #1805
